tests/unit_test/config.py

```diff
@@ -0,0 +1,292 @@
+    ):
+        s3_bucket = data_item["values"]["bucket"]
+        break
+
+s3_bucket_tags = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_s3_bucket" and data_item["name"] == "this":
+        s3_bucket_tags = data_item["values"]["tags"]
+        break
+
+vpc_cidr = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_vpc" and data_item["name"] == "this":
+        vpc_cidr = data_item["values"]["cidr_block"]
+        break
+
+vpc_public_subnet = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_subnet" and data_item["name"] == "public_subnet":
+        vpc_public_subnet = data_item["values"]["cidr_block"]
+        break
+
+vpc_private_subnet = None
+for data_item in combined_list:
+    if data_item["type"] == "aws_subnet" and data_item["name"] == "private_subnet":
+        vpc_private_subnet = data_item["values"]["cidr_block"]
+        break
+
+cognito = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cognito_user_pool"
+        and data_item["address"] == "module.xc3.aws_cognito_user_pool.grafana_pool[0]"
+    ):
+        cognito = data_item["values"]["name"]
+        break
+
+cloudwatch_list_linked_account = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "list_linked_accounts"
+    ):
+        cloudwatch_list_linked_account = data_item["values"]["name"]
+        break
+
+cloudwatch_list_linked_account_description = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "list_linked_accounts"
+    ):
+        cloudwatch_list_linked_account_description = data_item["values"]["description"]
+        break
+
+cloudwatch_list_linked_account_cron = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "list_linked_accounts"
+    ):
+        cloudwatch_list_linked_account_cron = data_item["values"]["schedule_expression"]
+        break
+
+cloudwatch_most_expensive_services = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "most_expensive_service"
+    ):
+        cloudwatch_most_expensive_services = data_item["values"]["name"]
+        break
+
+cloudwatch_most_expensive_services_description = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "most_expensive_service"
+    ):
+        cloudwatch_most_expensive_services_description = data_item["values"][
+            "description"
+        ]
+        break
+
+cloudwatch_most_expensive_services_cron = None
+for data_item in combined_list:
+    if (
+        data_item["type"] == "aws_cloudwatch_event_rule"
+        and data_item["name"] == "most_expensive_service"
```

Comment:
can we store it in variable and use that variable instead of passing it 3 times for each function

Reply:
Resolved in [d67f4bc](https://github.com/XgridInc/xc3/pull/18/commits/d67f4bcc757b95ad043c435e8c4acb4dd33c6a76)